packages/app-store/routing-forms/lib/getEventTypeAppMetadata.ts

```diff
@@ -0,0 +1,26 @@
+import { EventTypeMetaDataSchema } from "@calcom/prisma/zod-utils";
+
+import { enabledAppSlugs } from "./enabledApps";
+import type { Prisma } from ".prisma/client";
+
+const getEventTypeAppMetadata = (metadata: Prisma.JsonValue) => {
```

Comment:
This is used to get the app metadata from the event type for apps that are enabled for routing forms.